Sources/Basics/HTTPClient/HTTPClientResponse.swift

```diff
@@ -0,0 +1,57 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2020-2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+#if swift(<5.6)
+extension Data: UnsafeSendable {}
+#endif
+
+public struct HTTPClientResponse : Sendable {
```

Comment:
Moved out from `LegacyHTTPClient.swift` as a type reusable in both `LegacyHTTPClient` and `HTTPClient`.